torch_ttnn/passes/lowering/to_tt_guard.py

```diff
@@ -480,7 +480,7 @@
 # add output is bf16 type, which is wrong
 
 aten_slice_Tensor_blocklist += [
-    ["Tensor<[24]> self = ?", "int dim = 0", "Optional[int] start = 0", "Optional[int] end = 9223372036854775807"],
```

Comment:
why is this changed?

Reply:
Same as this https://github.com/tenstorrent/pytorch2.0_ttnn/pull/389#discussion_r1833651332